deluge/ui/console/commands/config.py

```diff
@@ -65,14 +65,14 @@ def simple_eval(source):
 class Command(BaseCommand):
     """Show and set configuration values"""
 
-    usage = "config [--set <key> <value>] [<key> [<key>...] ]"
+    usage = _("Usage: config [--set <key> <value>] [<key> [<key>...] ]")
```

Comment:
I wonder whether it's a good idea to translate strings like this which includes argument option description.
If `key` and `value` should be translated, they must also be translated in the other arguments below.